script.js

```diff
@@ -0,0 +1,69 @@
+function contain(title, search) {
+    let titleLw = title.toLowerCase();
+    let searchLw = search.toLowerCase();
+    return titleLw.includes(searchLw);
+  }
+  
+  function renderCourse(curCourse, parent, subString) {
```

Comment:
This function isn't correctly indented, while it's not required to be (as in Python) but it's considered a best practice to have your code formatted correctly.